4.7.1-windowsservercore-1709/build/Dockerfile

```diff
@@ -3,30 +3,34 @@ FROM microsoft/dotnet-framework:4.7.1-windowsservercore-1709
 SHELL ["powershell", "-Command", "$ErrorActionPreference = 'Stop'; $ProgressPreference = 'SilentlyContinue';"]
 
 # Install NuGet CLI
+ENV NUGET_VERSION 4.4.1
 RUN New-Item -Type Directory $Env:ProgramFiles\NuGet; \
-    Invoke-WebRequest -UseBasicParsing https://dist.nuget.org/win-x86-commandline/v4.4.1/nuget.exe -OutFile $Env:ProgramFiles\NuGet\nuget.exe
+    Invoke-WebRequest -UseBasicParsing https://dist.nuget.org/win-x86-commandline/v$Env:NUGET_VERSION/nuget.exe -OutFile $Env:ProgramFiles\NuGet\nuget.exe
```

Comment:
It is easy to miss the `v` in `/v$Env:NUGET_VERSION/`. How about including this character in the version variable as in `ENV NUGET_VERSION v4.4.1`? On the other hand, having a non-numeric in a version isn't better either.

Reply:
I had thought about this but decided against it.  The version is really just 4.4.1.  The download url is what uses the 'v' prefix.  Since this is an env var within the image I felt it was better to use the real version number.